mysql-test/suite/rocksdb_rpl/t/optimize_myrocks_replace_into.test

```diff
@@ -0,0 +1,38 @@
+--source include/have_rocksdb.inc
+--source include/master-slave.inc
+--source include/have_binlog_format_row.inc
+
+connection master;
+SET GLOBAL rocksdb_perf_context_level=3;
+
+create table t1(c1 int,c2 int, primary key (c1)) engine=rocksdb;
+insert into t1 values(1,1),(2,2),(3,3);
+select * from t1;
+
+
```

Comment:
Could you check GET_FROM_MEMTABLE_COUNT on slave as well? Something like this:

insert ...
```
connection slave;
select value into @c from information_schema.rocksdb_perf_context where table_schema='test' and table_name='t1' and stat_type='GET_FROM_MEMTABLE_COUNT';
connection master;
```
replace...
sync_slave_with_master;
```
select case when value-@c > 0 then 'false' else 'true' end as read_free from information_schema.rocksdb_perf_context where table_schema='test' and table_name='t1' and stat_type='GET_FROM_MEMTABLE_COUNT';
```